frontend/test/__support__/server-mocks/table.ts

```diff
@@ -20,3 +20,21 @@ export function setupTablesEndpoints(tables: Table[]) {
   fetchMock.get("path:/api/table", tables);
   tables.forEach(table => setupTableEndpoints(table));
 }
+
+export function setupUploadManagementEndpoint(tables: Table[]) {
+  fetchMock.get("path:/api/ee/upload-management/tables", tables);
+}
+
+/**
+ * mock the table deletion endpoint
+ * @param failureId - the id of the table that should fail to delete, any other id will succeed
+ */
+export function setupDeleteUploadManagementDeleteEndpoint(failureId?: number) {
+  fetchMock.delete(`glob:*/api/ee/upload-management/tables/*`, url => {
+    return url.includes(`/${failureId}`)
+      ? {
+          throws: { data: { message: "It's dead Jim" } },
```

Comment:
Ashamed to say I had to google this reference.

Reply:
A+ Code Review Thoroughness